### PR TITLE
Accordion added at /docs/index.md

### PR DESCRIPTION
**Pull request by aman-saggu-git**
I have a added Foundation Css supported Accordion to monoproject.com/docs/ page of mono website . It is added in /docs/index.md file

### CODE REVIEW

**Comment by Therzok** [comment]
Hey, can you please upload a screenshot of how it looks? Although, it's pretty ugly compared to Markdown (backend text wise).

**Comment by aman-saggu-git** [comment]
ok i am uploading a screenshot

**Comment by aman-saggu-git** [comment]
These are two screenshots for accordion 

PIC1

pic1

PIC2

pic2

**Comment by akoeplinger** [comment]
@bhallaheemanshu thanks for this suggestion and glad that you figured the git issues out :)

A few points:
- I don't like the blue panel wrapping the accordion items
- "Getting started" should be open by default when the page is loaded

All in all, I agree with @Therzok that the backend source looks pretty ugly compared to the original Markdown. I'm not sure we'll want to sacrifice that simplicity just for having the accordion.

Can you maybe try formatting the code a bit better with proper indentation, e.g. like this:

``` html
<dd> 
    <a><h2>Web</h2></a> 
    <div> 
        <ul> 
            <li><a href="/docs/web/aspnet/">ASP.NET</a></li> 
            <li><a href="/docs/web/wcf/">WCF</a></li>
        </ul>
    </div>
</dd>
```

**Comment by aman-saggu-git** [comment]
ok, i am doing proper formatting to code .and trying to find some solution regarding blue panel to make it simple

**Comment by aman-saggu-git** [comment]
- Blue Panels are Removed 
- "Getting Started" will open by default when page is loaded

Preview :

piv6

**Comment by aman-saggu-git** [comment]
what can I do for more simplicity to code

**Comment by akoeplinger** [comment]
This looks much better. It seems something went wrong with the second commit you pushed, since it reverts all the changes.

As for simplification: I think if it's properly formatted it'll be not that bad. Another option might be to keep the existing Markdown as-is and use jQuery to dynamically construct the required markup client side.

**Comment by aman-saggu-git** [comment]
sorry for that it is fixed now

**Comment by aman-saggu-git** [comment]
can i get any update about this request .

File: docs/index.md [hunk 1]
```diff
@@ -15,121 +15,199 @@ Our website is open source on [GitHub](https://github.com/mono/website). If you
...
- - [COM](/docs/advanced/com/)
-   - [COM-Interop](/docs/advanced/com-interop/)
- - [CoreCLR](/docs/advanced/coreclr/)
-   - [CoreCLR HowTo](/docs/advanced/coreclr-howto/)
- - [Performance Tips](/docs/advanced/performance-tips/)
- - [SafeHandles](/docs/advanced/safehandles/)
- - [Sandbox](/docs/advanced/sandbox/)
+<dl class="accordion" data-accordion>
+    <dd class="accordion-navigation">
+        <a class="panel" href="#panel1"><h2>Getting Started</h2></a>
```
Comment (akoeplinger) [inline docs/index.md]: The href (and the corresponding div) should be changed to a format like `href="#getting-started"` here and everywhere else to make the link look better.

Comment (aman-saggu-git) [inline docs/index.md]: ok

**Comment by akoeplinger** [comment]
@bhallaheemanshu I merged this PR in with https://github.com/mono/website/commit/0273c94f33707ef13c8854e059cc2d14635a5995, thanks for contributing!

**Comment by aman-saggu-git** [comment]
Thank you very much

**Pull request closed without merging**